odl/space/cu_ntuples.py

```diff
@@ -943,8 +1318,14 @@ def norm(self, x):
         norm : float
             The norm of the vector
         """
-        from math import sqrt
-        return sqrt(self.const) * float(_norm_default(x))
+        if self.exponent == 2.0:
+            from math import sqrt
+            return sqrt(self.const) * float(_norm_default(x))
+        elif self.exponent == float('inf'):  # Weighting irrelevant
```

Comment:
I'm not sure either. How about we make it throw an error in the `inf` case for now so people dont get used to a behaviour, and we make an issue on this to discuss it?